src/index.js

```diff
@@ -3,13 +3,7 @@
 /* eslint-disable import/no-mutable-exports */
 
 import './style.css';
-
-const myTodo = [
-  { description: 'Wash the dishes', completed: false, index: 0 },
-  { description: 'Clean the room', completed: false, index: 1 },
-  { description: 'Do the laundry', completed: false, index: 2 },
-  { description: 'Take out the trash', completed: false, index: 3 },
-];
+import myTodo from './modules/functionality.js';
```

Comment:
- [ ] Kindly fix this import statement since there's an error at the moment the web run

![image](https://user-images.githubusercontent.com/93630700/207457477-1428187d-8d65-4619-bdc8-64484c6e29cf.png)